src/cpp/session/SessionMain.cpp

```diff
@@ -239,6 +239,20 @@ using namespace rsession::client_events;
 namespace rstudio {
 namespace session {
 
+namespace {
+
+std::string s_fallbackLibraryPath;
+
+void onExit()
```

Comment:
Can the value we set in our executable actually persist after we exit? (i.e. do we actually need this code)